### PR TITLE
Add file size to json object && some tests

### PR DESCRIPTION
**Pull request by SimplyAhmazing**
Added a property that has a file/directory's size in bytes. I needed this for a project I'm working on, maybe others will want it too. Added tests.

### CODE REVIEW

**Comment by mihneadb** [comment]
Cool, thanks! Care to also set up a travis yml file for running those tests all the time? :)

**Comment by SimplyAhmazing** [comment]
Hey! I set up a travis yml file (first time I've done one of those) and I had to rejigger the tests. I also discovered and fixed a bug.

The `extensions` parameter didn't ignore 'dot files' so I added a part to the code that did that.

**Comment by mihneadb** [comment]
Sweet, thanks!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/SimplyAhmazing/node-directory-tree.svg)](https://travis-ci.org/SimplyAhmazing/node-directory-tree)
```
Comment (mihneadb) [inline README.md]: Please update this to the original URL of the repo :D.

**Comment by mihneadb** [comment]
Good to merge after that URL fix :). Thanks!

**Comment by SimplyAhmazing** [comment]
I fixed the url!

**Comment by mihneadb** [comment]
Sweet, thanks!

**Pull request merged**